src/foam/nanos/controller/ApplicationController.js

```diff
@@ -377,6 +378,19 @@ foam.CLASS({
       });
     },
 
+    function requestCapability(capabilityInfo) {
+      var self = this;
+      console.log("CAPABILITY SCREEN REQUESTED");
+      return new Promise(function(resolve, reject) {
+        self.stack.push({
+          class: 'foam.nanos.crunch.ui.CapabilityInterceptView',
+          data: self.__subContext__["capabilityDAO"],
```

Comment:
I think we can change this to

```suggestion
          data: self.capabilityDAO,
```

if we add `'capabilityDAO'` to the imports.